IBAnimatable/MaskDesignable.swift

```diff
@@ -50,7 +51,9 @@ public extension MaskDesignable where Self: UIView {
     case .triangle:
       maskTriangle()
     case .none:
-      layer.mask?.removeFromSuperlayer()
+      if previousMaskType != nil {
+        layer.mask?.removeFromSuperlayer()
```

Comment:
However... this would require either:
1) a class type protocol `public protocol MaskDesignable : class { ... }`
or
2) a mutating function `public extension MaskDesignable where Self: UIView {
  public func mutating configureMask() { ... }`